content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -1,47 +1,194 @@
+   a ticket is completed, and would have `Solved` as the :guilabel:`Reach Stage`. Simultaneously, an
+   :abbr:`SLA (Service Level Agreement)` titled `2 Days to Start` tracks the working time before
+   work on a ticket has begun, and would have `In Progress` as the :guilabel:`Reach Stage`.
+
+Time spent in stages selected in the :guilabel:`Excluding Stages` field are **not** included in the
+calculation of the :abbr:`SLA (Service Level Agreement)` deadline.
+
+Meet SLA deadlines
+==================
+
+As soon as it is determined that a ticket fits the criteria of an :abbr:`SLA (Service Level
+Agreement)` policy, a deadline is calculated. The deadline is based on the creation date of the
+ticket, and the targeted working hours.
+
+.. note::
+   The value indicated next to the :guilabel:`Working Hours` field of an :abbr:`SLA (Service Level
+   Agreement)` policy is used to determine the deadline. By default, this is determined by the value
+   set in the :guilabel:`Company Working Hours` field under :menuselection:`Settings app -->
+   Technical --> Resources --> Working Times`.
+
+The deadline is then added to the ticket, as well as a tag indicating the name of the :abbr:`SLA
+(Service Level Agreement)` applied.
+
+When a ticket satisfies an :abbr:`SLA (Service Level Agreement)` policy, the :abbr:`SLA (Service
+Level Agreement)` tag turns green, and the deadline disappears from view on the ticket.
+
+.. figure:: sla/sla-tags.png
    :align: center
-   :height: 330
-   :alt: View of a ticket’s form emphasizing a satisfied SLA in Odoo Helpdesk
+   :alt: A Helpdesk ticket with two SLA tags attached.
+
+   This Helpdesk ticket has two SLA policies applied to it. One of the policies has been satisfied,
+   so the tag has turned green. The other policy is in progress.
 
-SLA Analysis
-============
+.. important::
+   If a ticket fits the criteria for more than one :abbr:`SLA (Service Level Agreement)`, the
+   earliest occurring deadline is displayed on the ticket. After that deadline has passed, the next
+   deadline is displayed.
 
-Go to :menuselection:`Helpdesk --> Reporting --> SLA Status Analysis`. Apply *Filters* and
-*Group by* to identify tickets that should be prioritized and keep track of upcoming deadlines.
+   .. image:: sla/sla-deadline.png
+      :align: center
+      :alt: A Helpdesk ticket with emphasis on the deadline field.
 
-.. image:: sla/sla_analysis.png
+If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
+:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag turns red. After the
+:abbr:`SLA (Service Level Agreement)` has failed, the red tag stays on the ticket, even after the
+ticket is moved to the :guilabel:`Reach Stage`.
+
+Analyze SLA performance
+=======================
+
+The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
```

Comment:
```suggestion
The *SLA Status Analysis* report tracks how quickly an :abbr:`SLA (Service Level
```
I'm suggesting ** instead of GUI here...one, because, at this point, that title isn't appearing on the UI/screen (yet)...and, also, this is pretty much a definition